draft-ietf-core-echo-request-tag.md

```diff
@@ -174,7 +174,15 @@ Client   Server
 ~~~~~~~~~~
 {: #echo-figure title="Example Echo Option Message Flow" artwork-align="center"}
 
-When used to serve freshness requirements (including client aliveness and state synchronizing), CoAP messages containing the Echo option MUST be integrity protected between the intended endpoints, e.g. using DTLS, TLS, or an OSCORE Inner option ({{RFC8613}}). When used to demonstrate reachability at a claimed network address, the Echo option SHOULD contain the client's network address, but MAY be unprotected.
+Another way for the server to verify freshness is to maintain a cache of values associated to events: 
```

Comment:
I understand this to be a usage example. At this point I think it'd be easier for the reader to digest if this were simplified to a single latest-state (cache size 1), and explained in terms of "fresh" and "not fresh" rather than taking up the t0/t1 terminology of the time-based example.

Reply:
I like the addition but I think there are to much implementation details for this section. That kind of stuff we have in the appendix. I also think all time should be removed from the event based example.

One solution could be to udate the figure to

OLD     Echo: 0x437468756c687521 (t0)
NEW     Echo: 0x437468756c687521 (t0 or event value)

Alt.

OLD     Echo: 0x437468756c687521 (t0)
NEW     Echo: 0x437468756c687521 (t0 or e0)